docs/guides/email-sms.mdx

```diff
@@ -1,11 +0,0 @@
----
-id: email-sms
```

Comment:
When removing pages, please add a redirect to another page that is replacing this page. If no page is replacing it, redirect to home. You can do this here: https://github.com/ory/docs/blob/master/vercel.json#L1172

This is **very important**. When ever you change the URL of a page (change id, change slug, change directory name, delete a page), it **must** properly be rewritten. Please test the rewrite manually in the preview deployment for all pages you have modified the URL of.